docs/releases.rst

```diff
@@ -3,6 +3,78 @@
 Release notes
 =============
 
+Version 2.1.4
+-------------
+
+**New features**
+
+- Improve ``pg_rewind`` behavior on typical Debian/Ubuntu systems (Gunnar "Nick" Bluth)
+
+  On Debian/Ubuntu systems it is common to keep Postgres config files outside of the data directory. That makes ``pg_rewind --restore-target-wal`` to fail because it can't find ``postgresql.conf`` when trying to figure out``resolve_command``.
+
+- Allow setting ``TLSServerName`` on Consul service checks (Michael Gmelin)
+
+  Useful in case checks are done by IP and the Consul ``node_name`` is not an FQDN.
+
+- Added ``ppc64le`` support in watchdog (Jean-Michel Scheiwiler)
+
+  And fixed watchdog support on some non-x86 platforms.
+
+- Switched aws.py callback from ``boto`` to ``boto3`` (Alexander Kukushkin)
+
+  The ``boto`` is abandoned since 2018 and fails in python 3.9.
+
+- Periodically refresh service account token on K8s (Haitao Li)
+
+  Since Kubernetes v1.21 service account tokens expire in 1 hour.
+
+- Added ``/read-only-sync`` monitoring endpoint (Dennis4b)
+
+  It is similar to the ``/read-only`` but includes only synchronous replicas.
+
+
+**Stability improvements**
+
+- Don't copy the logical slot if there is a mismatch with the config (Alexander)
+
+  Previously Patroni was copying a logical slot to a replica with restart, noticing that ``plugin`` or ``database`` are different from the config and immediately removing it.
+
+- Special handling of recovery configuration parameters for PostgreSQL v12+ (Alexander)
+
+  While starting as replica Patroni is be able to update ``postgresql.conf`` and restart/reload if the leader address has changed by caching current parameters values instead of querying them from ``pg_settings``.
+
+- Better handling of IPv6 addresses in the ``postgresql.listen`` parameters (Alexander)
+
+  Since the ``listen`` parameter has a port people try to put IPv6 addresses into square braces which were not correctly stripped when there is more than one IP in the list.
```

Comment:
> Why does the presence of the port in the listen parameter suggest putting an IPv6 address into square brackets?

Because this is what people typically do when they want to make it clear that the last item isn't a hextet but port: `::1:8080` vs `[::1]:8080`.